Tests/SonarQube.TeamBuild.PreProcessor.Tests/Infrastructure/SampleExportXml.cs

```diff
@@ -15,10 +15,11 @@ internal static class SampleExportXml
         #region Example of XML exported by the Java plugin
 
         public const string RoslynExportedPluginKey = "csharp";
+        public const string RoslynStaticResourceName = "SonarLint.zip";
```

Comment:
Updated the example of XML returned by profile exporter to include the <Plugins> section and re-enabled the ignored part of the appropriate unit test.